impl/src/main/java/org/jboss/cdi/tck/tests/lookup/dynamic/DynamicLookupTest.java

```diff
@@ -236,4 +240,21 @@ public void testStream(Instance<Uncommon> uncommonInstance) {
         assertTrue(uncommonInstance.stream().filter(p -> p.getClass().equals(Garply.class)).findFirst().isPresent());
         assertTrue(uncommonInstance.stream().filter(p -> p.getClass().equals(Corge.class)).findFirst().isPresent());
     }
+
+    @Test
+    @SpecAssertions({ @SpecAssertion(section = BM_OBTAIN_INSTANCE, id = "a"), @SpecAssertion(section = BM_OBTAIN_INSTANCE, id = "b") })
+    public void beanManageCreateInstance() {
+        Instance<Object> instance = getCurrentManager().createInstance();
+        Instance<AsynchronousPaymentProcessor> asyncProcessors = instance.select(AsynchronousPaymentProcessor.class);
+        // there's no AsynchronousPaymentProcessor with default qualifier
```

Comment:
I think this is OK, but I would still prefer writing it the "positive" way, e.g. testing for presence of such bean rather than its absence. But that's just my feeling likely :)